pyfuncol/tests/test_no_forbiddenfruit.py

```diff
@@ -0,0 +1,7 @@
+from pyfuncol.list import *
```

Comment:
To make the import work you need to replace it with:

```suggestion
from ..list import *
```